### ISSUE TITLE
[Bug] Package dependency 

### ISSUE THREAD

**Issue by jason-chen-kiewit** [storybookjs/presets#249]
### Describe the bug

I've recently upgraded my project to React v18 and Storybook to v6.5.0-beta.0 and am currently unable to run `npm install @storybook/preset-create-react-app` in the terminal without getting dependency errors (see screenshot below). Hoping this will be a simple `peerDependencies` change, but any help is appreciated!

### Steps to reproduce the behavior

1. `npx create-react-app`
2. `npx sb init`
3. `npx sb upgrade --prerelease`
4. Delete `node_modules`
5. `npm install`
6. See error

### Expected behavior

`npm install`s without dependency errors.

### Screenshots and/or logs

![image](https://user-images.githubusercontent.com/79943064/165849240-eb2dcf5f-6150-4d2e-a593-eaa99ab39433.png)

### Environment

 - OS: Mac OS X
 - Node.js version: 17.6.0
 - NPM version: 8.7.0

### Additional context
N/A

**Comment by easga** [comment]
this issue is no more once the latest release (6.5.3) for storybook packages are used. Thanks for all those who made it happen, my storybook is revived :)

**Comment by SavePointSam** [comment]
Currently hitting this on fresh setup of Storybook using version 6.5.9 with preset version 4.1.2

**Comment by icebearlabs** [comment]
yep hitting this as well, throwing out @storybook/preset-create-react-app fixes the issue. Something in there is not right, yet.
